### ISSUE TITLE
No Windows Build

### ISSUE THREAD

**Issue by kienerj** [lich-uct/molpher-lib#7]
looks great guys!

Windows build would be well appreciated!

For the time being I suggest to mention this in the docs in the install section (On windows the conda command will say the package is not available which at first is confusing).

**Comment by martin-sicho** [comment]
Hi @kienerj!

glad you find the library helpful :)

Windows build would indeed be great and I am willing to work on it, but the time is sparse at the moment. But it definitely is on the top of the list at the moment. Thanks for the request! 

I at least fixed the documentation issue by updating the README file.

**Comment by martin-sicho** [comment]
The information about only Linux being supported at the moment is in the [install section of the docs](https://lich-uct.github.io/molpher-lib/v0.0.0b2/usage/installation.html), but should probably be made more visible as well.

**Comment by jaimergp** [comment]
Would you be interested in creating a multi-platform `conda-forge` recipe for this project? I see that there some dependencies that would need to be built too, but that should be doable. I am happy to help if you want!

**Comment by martin-sicho** [comment]
Oh definitely. The build process in general could use some improvements. Come see me if you're at the RDKit UGM :)
--

**Comment by martin-sicho** [comment]
@jaimergp Hi, it was great meeting you and having a chat in person! It is too bad we did not get to start working on this, but if you are still interested to help I definitely appreciate the input :) I really would like to add a Windows build in the near future and also do CI, coverage, etc... So let me know if you have a vision for this particular case or just some tips and tricks to share. Thanks!

And by the way, the dependencies might not even have to be built along with the library. The `CMakeLists.txt` has config options to find the dependencies in a conda environment automatically. The [current build script](https://github.com/lich-uct/molpher-lib/blob/master/conda/molpher-lib/molpher-lib/build.sh) actually uses the conda libraries rather than those installed in the project directory so getting and linking the dependencies on Windows should be easy. All you need to do to build the conda package on Linux now is just reduced to this:

```bash
git clone git@github.com:lich-uct/molpher-lib.git
cd conda

# this is really ugly and a relict of the past 
# i hope to replace it with just a call to `conda build molpher-lib`
# build.sh/build.bat should really do all the heavy lifting right?)
python build.py 
```

**Comment by jaimergp** [comment]
You are right! Are you interested in providing a package through `conda-forge`? In that case, fork https://github.com/conda-forge/staged-recipes and follow the instructions! Let me know if you need help along the way.

Your CMake configuration should help in getting this relatively painlessly :)

**Comment by martin-sicho** [comment]
Nice. I definitely would be interested in that. I will study that and post an update here on the progress. Thanks!
